test/test-remote/looker/src/args.ts

```diff
@@ -339,6 +371,13 @@ export function parseCmdlineArgs(): void {
     process.exit(1);
   }
 
+  if (CFG.metrics_past_start_range_min_seconds > CFG.metrics_past_start_range_max_seconds) {
+    log.error(
+      "metrics_past_start_range_min_seconds must not be larger than metrics_past_start_range_max_seconds"
+    );
+    process.exit(1);
+  }
```

Comment:
future: we should also warn (maybe not err) when this is too far in the past or in the future (and therefore known to not work with Cortex TSDB)

Reply:
added a warning in follow-up PR (was already in there and not difficult to add)